src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/servicebus/CreateUpdatesBusSender.java

```diff
@@ -1,44 +1,54 @@
 package uk.gov.hmcts.ethos.replacement.docmosis.servicebus;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
 import lombok.extern.slf4j.Slf4j;
+import org.postgresql.util.PGobject;
 import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.stereotype.Component;
 import uk.gov.hmcts.ecm.common.helpers.CreateUpdatesHelper;
 import uk.gov.hmcts.ecm.common.model.servicebus.CreateUpdatesDto;
 import uk.gov.hmcts.ecm.common.model.servicebus.CreateUpdatesMsg;
 import uk.gov.hmcts.ecm.common.model.servicebus.datamodel.DataModelParent;
 import uk.gov.hmcts.ecm.common.servicebus.ServiceBusSender;
+import uk.gov.hmcts.ethos.replacement.docmosis.service.FeatureToggleService;
 
+import java.sql.CallableStatement;
+import java.sql.Connection;
+import java.sql.SQLException;
 import java.util.List;
 import java.util.concurrent.atomic.AtomicInteger;
+import javax.sql.DataSource;
 
 /**
  * Sends create updates messages to create-updates queue.
  */
 @Slf4j
 @Component
 public class CreateUpdatesBusSender {
-
     private static final String ERROR_MESSAGE = "Failed to send the message to the queue";
     private final ServiceBusSender serviceBusSender;
+    private final FeatureToggleService featureToggleService;
+    private final DataSource dataSource;
 
     public CreateUpdatesBusSender(
-            @Qualifier("create-updates-send-helper") ServiceBusSender serviceBusSender) {
+            @Qualifier("create-updates-send-helper") ServiceBusSender serviceBusSender,
+            FeatureToggleService featureToggleService, DataSource dataSource) {
         this.serviceBusSender = serviceBusSender;
+        this.featureToggleService = featureToggleService;
+        this.dataSource = dataSource;
     }
 
-    public void sendUpdatesToQueue(CreateUpdatesDto createUpdatesDto, DataModelParent dataModelParent,
-                                   List<String> errors, String updateSize) {
+    public void sendMessageToServiceBus(CreateUpdatesDto createUpdatesDto, DataModelParent dataModelParent,
+            List<String> errors, String updateSize) {
         log.info("Started sending messages to create-updates queue");
 
         AtomicInteger successCount = new AtomicInteger(0);
 
-        List<CreateUpdatesMsg> createUpdatesMsgList =
-                CreateUpdatesHelper.getCreateUpdatesMessagesCollection(
-                        createUpdatesDto,
-                        dataModelParent,
-                        500,
-                        updateSize);
+        List<CreateUpdatesMsg> createUpdatesMsgList = CreateUpdatesHelper.getCreateUpdatesMessagesCollection(
+                createUpdatesDto,
+                dataModelParent,
+                500,
```

Comment:
Make this configurable please (but default to 500).

Reply:
This is old servicebus code, not touching this